tests/console/tcpdump.pm

```diff
@@ -0,0 +1,33 @@
+# SUSE's openQA tests
+#
+# Copyright 2025 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Package: tcpdump
+# Summary: test tcpdump by pinging a localhost and dumping with an icmp filter
+# Maintainer: QE Core <qe-core@suse.com>
+
+use base "consoletest";
+use strict;
+use warnings;
+use testapi;
+use utils;
+use serial_terminal 'select_serial_terminal';
+
+sub run {
+    my ($self) = @_;
+    my $tcpdump_log_file = "/tmp/tcpdump.log";
+    my $pid_file = '/tmp/tcpdump.pid';
+
+    select_serial_terminal;
+
+    zypper_call "in tcpdump";
+    # Start tcpdump to sniff only icmp loclhost packets in background and do ping
+    script_run("tcpdump -i lo icmp and src localhost -vv > $tcpdump_log_file 2>&1 & echo \$! > $pid_file & sleep 4");
+    assert_script_run("ping -c4 localhost -4 & sleep 4");
+
+    assert_script_run("kill \$(cat $pid_file)");
+    record_info("TEST LOG", script_output("cat $tcpdump_log_file"));
```

Comment:
Would you please upload the log in clean up stage and post fail hook?

Reply:
IMO that is not needed. there will be already the record_info output before validate_script_output

Reply:
Hmm, But we may miss this file if any failures before this step :)

Reply:
yes, but if ping or kill will fail then it's likely unrelated/test/infra issue